login-workflow/src/pages/create-account/steps/verify-email/verify-email.component.ts

```diff
@@ -40,13 +40,13 @@ import { BluiAuthTranslations } from '../../../../translations/auth-translations
                     </mat-error>
                 </mat-form-field>
             </form>
-            <button
-                mat-flat-button
+            <div style="display: flex; flex-direction: row;"><div class="mat-body-1">{{ translate.CREATE_ACCOUNT.VERIFY_EMAIL.RESEND_LABEL }}</div> 
+            <div
                 color="primary"
-                style="width: 100%; margin-top: 8px;"
+                class="blui-auth-link mat-body-1"
```

Comment:
use `mat-subheading-1` here instead of `mat-body-1`. We want that extra semibold font weight.

Reply:
Thanks @huayunh. Changed the class to make semibold font weight. see the screenshot
![image](https://user-images.githubusercontent.com/13012853/230129098-e34dcf00-702a-4f7b-bd52-d72eb9267f02.png)